### ISSUE TITLE
T and a overlapping

### ISSUE THREAD

**Issue by edmundlaugasson** [aardappel/treesheets#50]
If to write T and a side by side then they will overlap when activating presentation view (F12) then these letters will overlap. When to use small t then it will not overlap.

**Comment by aardappel** [comment]
I don't understand the exact configuration you're talking about, but that sounds like a font-specific issue that will be hard to fix generally.

**Comment by edmundlaugasson** [comment]
That is true - initially was Verdana, tried also Dejavu Sans, Arial - did not work. But Courier New, Bitstream Charter, *Mono, FreeSans, Nimbus Roman No9 L  are working. It seems that a bit condensed fonts do not work.
